alf/environments/gym_wrappers.py

```diff
@@ -464,100 +494,3 @@ def _pool_and_resize(self):
             interpolation=cv2.INTER_AREA)
         int_image = np.asarray(transformed_image, dtype=np.uint8)
         return np.expand_dims(int_image, axis=2)
```

Comment:
Seems all other frame-related wrappers (FrameResize,DMAtariPreprocessing) are still "channels_last". If FrameStack is not used, then there will be an issue

Reply:
ok, I implement both and put comments in suite_gym indicating how to use them.